Usage: celery [OPTIONS] COMMAND [ARGS]...
Try 'celery --help' for help.

Error: Invalid value for '-A' / '--app': 
Unable to load celery application.
While trying to load the module apps.worker.celery_app the following error occurred:
Traceback (most recent call last):
  File "/usr/local/lib/python3.8/dist-packages/celery/bin/celery.py", line 58, in convert
    return find_app(value)
  File "/usr/local/lib/python3.8/dist-packages/celery/app/utils.py", line 383, in find_app
    sym = symbol_by_name(app, imp=imp)
  File "/usr/local/lib/python3.8/dist-packages/kombu/utils/imports.py", line 59, in symbol_by_name
    module = imp(module_name, package=package, **kwargs)
  File "/usr/local/lib/python3.8/dist-packages/celery/utils/imports.py", line 109, in import_from_cwd
    return imp(module, package=package)
  File "/usr/lib/python3.8/importlib/__init__.py", line 127, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 1014, in _gcd_import
  File "<frozen importlib._bootstrap>", line 991, in _find_and_load
  File "<frozen importlib._bootstrap>", line 961, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "<frozen importlib._bootstrap>", line 1014, in _gcd_import
  File "<frozen importlib._bootstrap>", line 991, in _find_and_load
  File "<frozen importlib._bootstrap>", line 975, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 671, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 848, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/data/model_integration/apps/__init__.py", line 3, in <module>
    from apps.application import create_application
  File "/data/model_integration/apps/application.py", line 5, in <module>
    from apps.routers import setup_routers
  File "/data/model_integration/apps/routers/__init__.py", line 3, in <module>
    from apps.routers.v1 import router as v1_router
  File "/data/model_integration/apps/routers/v1/__init__.py", line 8, in <module>
    from apps.routers.v1.camera import router as camera_router
  File "/data/model_integration/apps/routers/v1/camera.py", line 15, in <module>
    from apps.services.camera import VideoTaskServer
  File "/data/model_integration/apps/services/camera.py", line 5, in <module>
    from apps.worker.celery_worker import start_video_task
  File "/data/model_integration/apps/worker/celery_worker.py", line 13, in <module>
    from apps.detection.YOLOv5_detector import YOLOv5Detector
  File "/data/model_integration/apps/detection/YOLOv5_detector.py", line 5, in <module>
    from myutils import is_bbox_partially_inside_region
ModuleNotFoundError: No module named 'myutils'

Traceback (most recent call last):
  File "/usr/local/bin/uvicorn", line 8, in <module>
    sys.exit(main())
  File "/usr/local/lib/python3.8/dist-packages/click/core.py", line 1157, in __call__
    return self.main(*args, **kwargs)
  File "/usr/local/lib/python3.8/dist-packages/click/core.py", line 1078, in main
    rv = self.invoke(ctx)
  File "/usr/local/lib/python3.8/dist-packages/click/core.py", line 1434, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/usr/local/lib/python3.8/dist-packages/click/core.py", line 783, in invoke
    return __callback(*args, **kwargs)
  File "/usr/local/lib/python3.8/dist-packages/uvicorn/main.py", line 416, in main
    run(
  File "/usr/local/lib/python3.8/dist-packages/uvicorn/main.py", line 587, in run
    server.run()
  File "/usr/local/lib/python3.8/dist-packages/uvicorn/server.py", line 61, in run
    return asyncio.run(self.serve(sockets=sockets))
  File "/usr/lib/python3.8/asyncio/runners.py", line 44, in run
    return loop.run_until_complete(main)
  File "/usr/lib/python3.8/asyncio/base_events.py", line 616, in run_until_complete
    return future.result()
  File "/usr/local/lib/python3.8/dist-packages/uvicorn/server.py", line 68, in serve
    config.load()
  File "/usr/local/lib/python3.8/dist-packages/uvicorn/config.py", line 467, in load
    self.loaded_app = import_from_string(self.app)
  File "/usr/local/lib/python3.8/dist-packages/uvicorn/importer.py", line 24, in import_from_string
    raise exc from None
  File "/usr/local/lib/python3.8/dist-packages/uvicorn/importer.py", line 21, in import_from_string
    module = importlib.import_module(module_str)
  File "/usr/lib/python3.8/importlib/__init__.py", line 127, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 1014, in _gcd_import
  File "<frozen importlib._bootstrap>", line 991, in _find_and_load
  File "<frozen importlib._bootstrap>", line 975, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 671, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 848, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/data/model_integration/apps/__init__.py", line 3, in <module>
    from apps.application import create_application
  File "/data/model_integration/apps/application.py", line 5, in <module>
    from apps.routers import setup_routers
  File "/data/model_integration/apps/routers/__init__.py", line 3, in <module>
    from apps.routers.v1 import router as v1_router
  File "/data/model_integration/apps/routers/v1/__init__.py", line 8, in <module>
    from apps.routers.v1.camera import router as camera_router
  File "/data/model_integration/apps/routers/v1/camera.py", line 15, in <module>
    from apps.services.camera import VideoTaskServer
  File "/data/model_integration/apps/services/camera.py", line 5, in <module>
    from apps.worker.celery_worker import start_video_task
  File "/data/model_integration/apps/worker/celery_worker.py", line 13, in <module>
    from apps.detection.YOLOv5_detector import YOLOv5Detector
  File "/data/model_integration/apps/detection/YOLOv5_detector.py", line 5, in <module>
    from myutils import is_bbox_partially_inside_region
ModuleNotFoundError: No module named 'myutils'
